pkg/controller/pxcrestore/manager.go

```diff
@@ -0,0 +1,259 @@
+package pxcrestore
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/k8s"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc/backup"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc/backup/storage"
+)
+
+// TODO: choose a better name for this interface
```

Comment:
I'd like to change it too since Manager is also a term used in controller-runtime but couldn't find a good alternative. @inelpandzic ?

Reply:
I agree that manager is not good, what about `Restorer`? With it we could have `RestorerOptions`

Reply:
6fd564a